docs(README): make intro more concise (#1638)
lumen-oss/lux: README.md

## :star2: Key Features

#### :package: Create and Manage Lua Projects

- Easily manage dependencies, build steps and more through the `lux.toml` file.
- Full support for existing `.rockspec` files.

#### :gear: Modern, Comfortable Tooling

- Parallel builds and installs.
- Add/remove dependencies with simple CLI commands.
- Integrated code formatting, type checking and powerful code linting.

> - Good DX - automatically adds project dependencies to [`.luarc.json`](https://luals.github.io/wiki/configuration/#luarcjson-file) for proper LSP completions.

Reviewer: ```suggestion
- Good DX - automatically adds project dependencies to [`.luarc.json`](https://luals.github.io/wiki/configuration/#luarcjson-file) for proper LSP completions and type checking.
```

Author: we already mention type checking above, so to prevent ourselves from repeating perhaps:
```suggestion
- Good DX - automatically adds project dependencies to [`.luarc.json`](https://luals.github.io/wiki/configuration/#luarcjson-file) for rich LSP diagnostics.
```

#### :tada: Automatic Generation of Rockspecs

- Say goodbye to managing 10 different rockspec files in your source code!

#### :jigsaw: Lua Versioning Done Easy

- Lux takes care of installing the right versions of Lua headers automatically.
- Forget about users complaining they have the wrong Lua installed on their system.

#### :lock: Advanced Lockfile Support

> - Makes for fully reproducible developer environments.

Reviewer: suggestion: no repetition 

```suggestion
- Fully reproducible developer environments.
```

- Makes Lux easy to integrate with Nix!

#### :globe_with_meridians: Fully Compatible with the Ecosystem

- Works with existing LuaRocks packages.
- Have a complex rockspec that you don't want to rewrite to TOML? No problem!
  Lux allows the creation of an `extra.rockspec` file, everything just works.
- Have a very complex build script? Lux can shell out to `luarocks` if it detects it has
  to preserve maximum compatibility.

## :gear: How To Use

Feel free to consult the [documentation](https://lux.lumen-labs.org/tutorial/getting-started) on how to get started with Lux!

It features a tutorial and several guides to make you good at managing Lua projects.

> > [!WARNING]

Reviewer: suggestion: The warning could deter users. Perhaps it's time to change it to

```suggestion
> [!NOTE]
```
